fontbe/src/error.rs

```diff
@@ -72,7 +72,7 @@ pub enum Error {
     #[error("No glyph class '{0}'")]
     MissingGlyphClass(GlyphName),
     #[error("Multiple assignments for class: {0:?}")]
-    PreviouslyAssignedClass(PreviouslyAssignedClass),
+    PreviouslyAssignedClass(#[from] PreviouslyAssignedClass),
```

Comment:
Optional, to me naming closer to the error string (MultipleAssignmentsForClass) would be more intuitive

Reply:
sure, can do that as followup.